File: Data Structures and Algorithms/Programiz/Algorithms/Searching Algorithms/Linear Search/linear/linear-search.py
# Linear serach in python
def linearSearch(array, number, length):
    # go through array sequentially
    for i in range(0, length):
        if (array[i] == number):
            return i
    return -1

num = 1
array = [2, 4, 6, 8, 1, 10]
array_len = len(array)

result = linearSearch(array, num, array_len)

if result == -1:
    print("Element not found.")
else:
    print(f"Element found at index: {result}")

File: Data Structures and Algorithms/Programiz/Algorithms/Searching Algorithms/Linear Search/linear/linear-search.c
// Linear Search in C
#include <stdio.h>

int search(int array[], int n, int x) {
    // go through array sequencially
    for (int i=0; i<n; i++)
        if (array[i] == x)
            return i;
    return -1;
}

int main() {
    int x = 1;
    int array[] = {2,4,0,1,49};
    int n = sizeof(array) / sizeof(array[0]);
    //printf("n: %d", n);

    int result = search(array, n, x);

    (result == -1) ? printf("Element not found"): printf("Element found at index: %d", result);
}